src/react-components/inputs/input-with-suggestions.jsx

```diff
@@ -63,15 +74,25 @@ const InputWithSuggestions = ({ data, onClick, sortFunction, displayValue, ...pr
 
     return (
     <>
-    <Input
-        {...props}
-        onChange={onChange}
-        value={inputValue}
-        onFocus={() => {setTouched(true)}}
-    />
-    <div>
-        { touched && resultsToShow }
-    </div>
+        <div style={{display: 'flex', gap: '10px', 'alignItems': 'center'}}>
+            <Input
+                {...props}
+                onChange={onChange}
+                value={inputValue}
+                onFocus={() => {setTouched(true)}}
+                placeholder={placeholder ? placeholder : 'Select an Option'}
```

Comment:
placeholder={placeholder ? placeholder : 'Select an Option'} -> placeholder={placeholder || 'Select an Option'}
I think this shorten expression should work. I tested on browser's snippets with placeholder as '', null and undefined. 
Please check this if you have time.